### PR TITLE
Update README, script-hscs-smart-contract, script-transfer-hbar

### PR DESCRIPTION
**Pull request by theekrystallee**

### CODE REVIEW

File: transfer-hbar/script-transfer-hbar.js [hunk 1]
```diff
@@ -63,8 +63,8 @@ async function scriptTransferHbar() {
     logger.log('The transfer transaction ID:', transferTxId.toString());
 
     // Sign the transaction with the account that is being debited (operator account) and the transaction fee payer account (operator account)
-    // Since the account that is being debited and the account that is paying for the transaction are the same only one accoun'ts signature is required
-    const transferTxSigned = await transferTx.sign(operatorKey);
+    // Since the account that is being debited and the account that is paying for the transaction are the same, only one account's signature is required
+for transferTxSigned = await transferTx.sign(operatorKey);
```
Comment (bguiz) [inline transfer-hbar/script-transfer-hbar.js]: replacing `const` with `for` here is likely to break the script. have you tried running it?

Comment (theekrystallee) [reply transfer-hbar/script-transfer-hbar.js]: oops, not sure why that is there. should just be `const`

Comment (theekrystallee) [reply transfer-hbar/script-transfer-hbar.js]: updated. thanks for catching this!

**Review by bguiz — APPROVED** [review]

**Review by bguiz — APPROVED** [review]
Will merged this into the same branch as https://github.com/hedera-dev/hello-future-world-js/pull/12 so that it can be tested together.

**Comment by bguiz** [comment]
> Will merged this into the same branch as #12 so that it can be tested together.

Will merge*

**Pull request merged**